lib/signaling/v2/peerconnection.js

```diff
@@ -1717,6 +1749,11 @@ function updateEncodingParameters(pcv2) {
       params.encodings[0].networkPriority = 'high';
     }
 
+    if (!isFirefox && sender.track.kind === 'video') {
```

Comment:
I don't think the `isFirefox` check is needed, because there is only one encoding, and so the logic will set `scaleResolutionDownBy` to 1, which is what we want.
```suggestion
    if (sender.track.kind === 'video') {
```

Reply:
I noticed that firefox does not like these changes. if we do set `scaleResolutionDownBy` on firefox, it fails to apply SDP on subsequent negotiation.